src/styles/elements/Heading.js

```diff
@@ -2,7 +2,7 @@ import styled from 'styled-components';
 
 import { colors, effects, spacing } from '../base/base';
 
-export const H1 = styled.h1`
+export const Heading = styled.p`
```

Comment:
I think, that it could stay as `h1` as a default, or `h2`, to not make a mistake and put more h1 than one on the page